test/acceptance/pages/WhatIsYourEmailPage.scala

```diff
@@ -19,16 +19,18 @@ package acceptance.pages
 import acceptance.utils.Configuration
 import org.openqa.selenium.By
 
-class WhatIsYourEmailPage extends BasePage {
-  override val url: String = Configuration.frontendHost + "/customs-email-frontend/change-email-address"
-  override val title = "Change your email address for CDS"
-
+class EmailPage extends BasePage {
```

Comment:
Changing this to be an `abstract` class, would avoid the need to have empty string for `title` and `url`. You should be able to remove following overrides and still maintain the pattern of class (abstract in this case) with companion object

Reply:
changed to an abstract class